stories/__tests__/SQFormDropdown.stories.test.js

```diff
@@ -23,6 +23,12 @@ it('should render with empty initial value', () => {
   expect(labelValue).toHaveTextContent('- -');
 });
 
+/* There's not a way for me to test non-empty intiial values
```

Comment:
This is a limitation caused by how we have to construct the stories. The component has to be wrapped in SQForm for it to work, and the storybooks args have to go on the target component for the storybook stuff to work. That means we don't really have a straight forward way to pass anything to the component's parent (SQForm).

cc @SeanGroff 

We could probably pass in a custom arg like `args.SQFormInitialValues` to SQForm's `initialValues` prop, but I feel like that's going to start complicating the stories because we're trying to make them do something they're not really meant to do. If we did this there may also be the overhead of keeping track of which storybook args are meant for the component and which are meant for SQForm for the purpose of testing.

I think some more discovery is needed to figure out if testing stories is really the right way to go for this library

Reply:
I don't have my head fully wrapped around this BUT my fire from the hip reaction is we need to supplement our integration tests with Cypress e2e tests in Storybook. 

Or maybe it's the inverse, we create Cypress e2e tests and supplement the e2e tests with these integration tests.